Cargo.toml

```diff
@@ -56,6 +56,7 @@ fugit = "0.3"
 rtt-logger = "0.2"
 systick-monotonic = "1.0"
 mono-clock = "0.1"
+spinning = { version = "0.1", default-features = false }
```

Comment:
I wonder whether the more active `spin` might be a safer bet here. `spinning` seems a bit dead and claims to distinguish itself from `spin` by features that `spin` certainly has (`lock_api` etc). Certainly compiles fine with `spin` instead.

Reply:
Didn't even see `spin`. Happy to move to a more maintained crate!